src/modules/feed/FeedPage.module.scss

```diff
@@ -1,22 +1,95 @@
-.root {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  text-align: center;
-}
-
-.page {
-  padding: 2em;
-  display: flex;
-  height: 100vh;
-}
-
-.form {
-  margin: auto;
-  width: 100%;
-  max-width: 32em !important;
-}
-
-.submit_button {
-  width: 100%;
+:global(#root) {
+  .root {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+  }
+
+  .container {
+    display: flex;
+    flex-direction: column;
+    overflow: hidden;
+  }
+
+  .create_post_container_mobile {
+    padding: 16px;
+  }
+
+  .create_post_segment {
+    width: 480px;
+    margin: auto;
+    padding-left: 30px;
+    padding-right: 30px;
+    border-radius: 18px;
+    border: none;
+  }
+
+  .divider {
+    margin-bottom: 2px;
+    margin-top: 0;
+  }
+
+  .form {
+    margin: auto;
+    width: 100%;
+    max-width: 512px;
+  }
+
+  .board_container {
+    max-width: 480px;
+    -webkit-overflow-scrolling: touch;
```

Comment:
Small nit: if you use prefixed properties, leave a quick comment why this is only needed for `-webkit`